### DISCUSSION TITLE
Subsetting SCENIC+ based on sample ID

### DISCUSSION THREAD

**Discussion by Priyadarshini-Rai** [aertslab/scenicplus#135] [Q&A]
Can we subset the SCENIC+ object based on the cells belonging to a particular sample, if we have multiple samples in the data?

**Comment by SeppeDeWinter** [comment]
Hi @Priyadarshini-Rai 

Yes you can, see: https://scenicplus.readthedocs.io/en/latest/api.html#scenicplus.scenicplus_class.SCENICPLUS.subset

However this can only be done _before_ running the SCENIC+ analysis.

Best,

Seppe
